.gitlab/notify/notify.yml

```diff
@@ -64,8 +64,13 @@ notify_github:
     - !reference [.except_mergequeue]
     - !reference [.except_main_or_release_branch]
     - !reference [.except_no_tests_no_deploy]
-    - !reference [.on_a7]
-    - when: on_success
+    - if: $RELEASE_VERSION_7 != ""
```

Comment:
@amenasria is it better like this ? I'm not sure I can merge more properly .on_a7 and changes